src/custom/state/lists/hooks/hooksMod.ts

```diff
@@ -171,9 +180,11 @@ export function useInactiveListUrls(): string[] {
   const chainId = supportedChainId(connectedChainId) ?? DEFAULT_NETWORK_FOR_LISTS
   const lists = useAllLists()
   const allActiveListUrls = useActiveListUrls()
-  return Object.keys(lists).filter(
-    (url) => !allActiveListUrls?.includes(url) && !UNSUPPORTED_LIST_URLS[chainId].includes(url)
-  )
+  return useMemo(() => {
+    return Object.keys(lists).filter(
+      (url) => !allActiveListUrls?.includes(url) && !UNSUPPORTED_LIST_URLS[chainId].includes(url)
+    )
+  }, [chainId, allActiveListUrls, lists])
```

Comment:
Sasha has started using a trick to avoid re-renders and to deal with the issue of object comparison in hooks deps that may or may not help here:

Stringify the object dependency and make that the hook dependency instead.
Then ignore the rule of hooks linter rule and use `lists` as usual

Reply:
yes, that's something that can be done taking into account:
1. we should identify why a dependency reference changes and go up on the hierarchy
2. the size of the object. For smaller objects that wouldn't be a problem but for big ones like the tokens list it would

This case was a low hanging fruit fix as the hook was always generating a new array and the callers that used this list as a dependency were always re-rendering